client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -203,6 +206,16 @@ const Organizations = () => {
           </LinkItem>
         ))}
       </List>
+      <DownloadUserListButton
+        icon="download"
+        href="/api/support/organizations/users"
+      >
+        Download User List
+      </DownloadUserListButton>
+      <p>
+        Export a list of Audit Admins and Jurisdiction Managers for recently
```

Comment:
I wonder if instead of `recently` it should explicitly say `within the last 12 weeks`, so there is no ambiguity if someone is confused why an audit from 3.5 months back isn't showing

Reply:
Good point, updated in [0be290c](https://github.com/votingworks/arlo/pull/2077/commits/0be290c47651edf8421bdd57b53177bee6ac8d46)